docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
-
-Once powered, the PC will boot from the USB key and will automatically get to a graphical user interface with an opened web browser prompting for PacketFence configuration. The configuration process is a five steps process at the end of which, the USB key will be a persistent working PacketFence environment.
-
-Step 1: Enforcement
-+++++++++++++++++++
-The first and most important step of the configuration process. This is where you'll choose the enforcement technique; either VLAN (out-of-band), INLINE (in-band) or WebAuth.
-
-The choice(s) made on this step will influence the next step where you'll need to configure the different networks.
-
-Each enforcement mode has its own required interface types that you'll have to configure on step 2.
-
-VLAN enforcement
-^^^^^^^^^^^^^^^^
-
-Step 2: Networks
-++++++++++++++++
-This step will ask you to statically configure your network interfaces (note that DHCP interfaces configuration is not supported yet).
-
-Depending on the choice(s) made on step 1, you'll have to configure the required types of interface. The web interface will list all currently installed network interfaces on the system. An IP and a netmask will be visible if the network interface is configured (either by DHCP or already manually configured). You can edit those ones, create/delete VLANs on physical interfaces and enable/disable an interface. Note that theses changes are effective on the moment you make them. Persistence will be written only for ENABLED interfaces.
-
-In all time, you'll need to set a Management interface.
-
-Required interface types for inline enforcement:
-
-    Management
-    Inline
-
-Required interface types for VLAN enforcement:
-
-    Management
-    Registration
-    Isolation
-
-Note that you can only set ONE (1) management interface. This one will work for both in the case you choose both modes.
-
-In our customer scenario, you will create two new vlans on the wired interface (will be eth0 most of the time).
-To do so, click the 'Add VLAN' button besides the wired interface for each of the needed vlan:
-
-Here's a sample configuration for both of them:
-
-Registration
-
-    Virtual LAN ID: 2
-    IP Address: 192.168.2.1
-    Netmask: 255.255.255.0
-
-Isolation
-
-    Virtual LAN ID: 3
-    IP Address: 192.168.3.1
-    Netmask: 255.255.255.0
-
-Don't forget to also edit the physical interface with the correct management network information by clicking the 'Edit' button next to it.
-
-According to our customer scenario, you will associate the correct type the each interfaces.
-
-    eth0: Management
-    eth0 VLAN 2: Registration
-    eth0 VLAN 3: Isolation
+Step 1: Set-Up the Aerohive APs and SSID
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Make sure that those three (3) interfaces are in an Enabled state for the persistence to occur.
+Logon to your Hive Manager interface::
```

Comment:
It's called "HiveManager" - make sure you use that everywhere - not HM, "Hive Manager" "Hive manager"